src/import.cmd.js

```diff
@@ -52,9 +53,9 @@ export default class ImportCommand extends AbstractServerCommand {
         depth: 1,
         singleBranch: true,
       });
-      this.log.info('Franklin Importer UI is ready.');
+      this.log.info(`AEM Importer UI is ready. v${await getUIVersion()}`);
     } else {
-      this.log.info('Fetching latest version of the Franklin Import UI.');
+      this.log.info('Fetching latest version of the AEM Import UI...');
```

Comment:
```suggestion
      this.log.info('Fetching latest version of the AEM Importer UI...');
```

consistent with the line above.